cpp/benchmarks/join/mixed_join.cu

```diff
@@ -149,6 +200,16 @@ NVBENCH_BENCH_TYPES(nvbench_mixed_inner_join,
   .add_int64_axis("left_size", JOIN_SIZE_RANGE)
   .add_int64_axis("right_size", JOIN_SIZE_RANGE);
 
+NVBENCH_BENCH_TYPES(nvbench_mixed_inner_join_complex_ast,
+                    NVBENCH_TYPE_AXES(JOIN_NULLABLE_RANGE,
+                                      DEFAULT_JOIN_NULL_EQUALITY,
+                                      DEFAULT_JOIN_DATATYPES))
+  .set_name("mixed_inner_join_complex_ast")
+  .set_type_axes_names({"Nullable", "NullEquality", "DataType"})
+  .add_int64_axis("left_size", JOIN_SIZE_RANGE)
+  .add_int64_axis("right_size", JOIN_SIZE_RANGE)
+  .add_int64_axis("ast_operators", {1, 5, 10});
```

Comment:
I think the number of operators is really `2 * ast_operators - 1`. Maybe we should name it "AST tree depth" or something like that.

Reply:
Good point. Renamed it to `ast_levels` to stay consistent with the naming used in the transform benchmarks.